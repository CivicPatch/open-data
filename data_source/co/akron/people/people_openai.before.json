[
  {
    "name": "Brandon Hill",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "Jared Jefferson",
    "roles": [
      "Trustee",
      "Mayor Pro Tem"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2028",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "Crystann Benson",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2028",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "Braden Brent",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2028",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "Ariella Gonzales-Vondy",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "Jennifer Hansen",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  },
  {
    "name": "David Kembel",
    "roles": [
      "Trustee"
    ],
    "divisions": [],
    "image": null,
    "phone_number": null,
    "email": null,
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://townofakron.colorado.gov/government/board-of-trustees"
    ],
    "updated_at": "2025-07-09T00:14:19Z"
  }
]