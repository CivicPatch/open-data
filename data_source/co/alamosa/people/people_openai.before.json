[
  {
    "name": "Jackie Vigil",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 3"
    ],
    "image": "https://cityofalamosa.org/wp-content/uploads/2022/02/JackieVigil-683x1024.jpg",
    "phone_number": "(719) 992-8008",
    "email": null,
    "website": "https://cityofalamosa.org/mayorprotem-ward3",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/mayorprotem-ward3",
      "https://cityofalamosa.org/team/city-council"
    ],
    "updated_at": "2025-06-30T17:01:01Z"
  },
  {
    "name": "Ty Coleman",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": "https://cityofalamosa.org/wp-content/uploads/2022/02/IMG_0479-683x1024.jpg",
    "phone_number": "(719) 313-7241",
    "email": null,
    "website": "https://cityofalamosa.org/mayor",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/mayor",
      "https://cityofalamosa.org/team/city-council"
    ],
    "updated_at": "2025-06-30T17:01:01Z"
  },
  {
    "name": "Jan Vigil",
    "roles": [
      "Council Member At-Large"
    ],
    "divisions": [
      "At-Large"
    ],
    "image": "https://cityofalamosa.org/wp-content/uploads/2022/02/IMG_0490-683x1024.jpg",
    "phone_number": "(970) 946-5611",
    "email": null,
    "website": "https://cityofalamosa.org/councilor-at-large-vigil",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/councilor-at-large-vigil",
      "https://cityofalamosa.org/team/city-council"
    ],
    "updated_at": "2025-06-30T17:01:01Z"
  },
  {
    "name": "Dawn Krebs",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 2"
    ],
    "image": "https://cityofalamosa.org/wp-content/uploads/2022/02/IMG_9105-683x1024.jpg",
    "phone_number": "(719) 580-2194",
    "email": null,
    "website": "https://cityofalamosa.org/city-councilor-ward-2",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/city-councilor-ward-2",
      "https://cityofalamosa.org/team/city-council"
    ],
    "updated_at": "2025-06-30T17:01:01Z"
  },
  {
    "name": "Michael Carson",
    "roles": [
      "Council Member At-Large"
    ],
    "divisions": [
      "At-Large"
    ],
    "image": "https://cityofalamosa.org/wp-content/uploads/2022/02/IMG_9183.jpg",
    "phone_number": "(719) 589-2593",
    "email": null,
    "website": "https://cityofalamosa.org/councilor-at-large-daniel",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/councilor-at-large-daniel"
    ],
    "updated_at": "2025-06-30T17:01:01Z"
  },
  {
    "name": "Liz Hensley",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 1"
    ],
    "image": "https://cityofalamosa.org/wp-content/uploads/2022/02/IMG_0486.jpg",
    "phone_number": "(719) 580-0201",
    "email": null,
    "website": "https://cityofalamosa.org/city-councilor-ward-1",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/city-councilor-ward-1"
    ],
    "updated_at": "2025-06-30T17:01:01Z"
  },
  {
    "name": "Jamie Dominguez",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 4"
    ],
    "image": "https://cityofalamosa.org/wp-content/uploads/2022/02/JamieDominguez-scaled.jpg",
    "phone_number": "(719) 937-3039",
    "email": null,
    "website": "https://cityofalamosa.org/city-councilor-ward-4",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://cityofalamosa.org/team/city-council",
      "https://cityofalamosa.org/city-councilor-ward-4"
    ],
    "updated_at": "2025-06-30T17:01:01Z"
  }
]