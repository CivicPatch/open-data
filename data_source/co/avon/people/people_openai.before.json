[
  {
    "name": "Tamra Underwood",
    "roles": [
      "Mayor",
      "Mayor Pro Tem"
    ],
    "divisions": [],
    "image": "https://www.avon.org/ImageRepository/Document?documentId=22948",
    "phone_number": null,
    "email": "tunderwood@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=163",
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=163"
    ],
    "updated_at": "2025-06-30T17:33:08Z"
  },
  {
    "name": "Rich Carroll",
    "roles": [
      "Mayor Pro Tem"
    ],
    "divisions": [],
    "image": "https://www.avon.org/ImageRepository/Document?documentID=24984",
    "phone_number": null,
    "email": "rcarroll@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=7",
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=7"
    ],
    "updated_at": "2025-06-30T17:33:08Z"
  },
  {
    "name": "Chico Thuon",
    "roles": [
      "Town Council Member"
    ],
    "divisions": [],
    "image": "https://www.avon.org/ImageRepository/Document?documentID=19238",
    "phone_number": null,
    "email": "cthuon@avon.org",
    "website": null,
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council"
    ],
    "updated_at": "2025-06-30T17:33:08Z"
  },
  {
    "name": "Lindsay Hardy",
    "roles": [
      "Town Council Member",
      "Council Member"
    ],
    "divisions": [],
    "image": "https://www.avon.org/ImageRepository/Document?documentId=24973",
    "phone_number": null,
    "email": "lhardy@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=204",
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=204"
    ],
    "updated_at": "2025-06-30T17:33:08Z"
  },
  {
    "name": "Ruth Stanley",
    "roles": [
      "Town Council Member",
      "Council Member"
    ],
    "divisions": [],
    "image": "https://www.avon.org/ImageRepository/Document?documentId=22947",
    "phone_number": null,
    "email": "rstanley@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=339",
    "start_date": "2022",
    "end_date": "2026",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=339"
    ],
    "updated_at": "2025-06-30T17:33:08Z"
  },
  {
    "name": "Gary Brooks",
    "roles": [
      "Town Council Member",
      "Council Member"
    ],
    "divisions": [],
    "image": "https://www.avon.org/ImageRepository/Document?documentId=24748",
    "phone_number": null,
    "email": "gbrooks@avon.org",
    "website": "https://www.avon.org/directory.aspx?EID=373",
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council",
      "https://www.avon.org/directory.aspx?EID=373"
    ],
    "updated_at": "2025-06-30T17:33:08Z"
  },
  {
    "name": "Kevin Hyatt",
    "roles": [
      "Town Council Member"
    ],
    "divisions": [],
    "image": "https://www.avon.org/ImageRepository/Document?documentId=24747",
    "phone_number": null,
    "email": "khyatt@avon.org",
    "website": null,
    "start_date": "2024",
    "end_date": "2028",
    "sources": [
      "https://www.avon.org/141/Avon-Town-Council"
    ],
    "updated_at": "2025-06-30T17:33:08Z"
  }
]