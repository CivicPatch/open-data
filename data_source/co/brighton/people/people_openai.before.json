[
  {
    "name": "Greg Mills",
    "roles": [
      "Mayor"
    ],
    "divisions": [
      "At-Large",
      "Ward 2"
    ],
    "image": "https://co-brighton2.civicplus.com/ImageRepository/Document?documentID=13275",
    "phone_number": "303-655-2268",
    "email": "gmills@brightonco.gov",
    "website": "https://co-brighton2.civicplus.com/1195/Greg-Mills",
    "start_date": "2022-01",
    "end_date": "2026-01",
    "sources": [
      "https://www.brightonco.gov/203/City-Council",
      "https://co-brighton2.civicplus.com/1195/Greg-Mills"
    ],
    "updated_at": "2025-06-27T20:53:33Z"
  },
  {
    "name": "Tom Green",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 1"
    ],
    "image": "https://www.brightonco.gov/ImageRepository/Document?documentID=29817",
    "phone_number": "303-655-2261",
    "email": "tgreen@brightonco.gov",
    "website": "https://www.brightonco.gov/2872",
    "start_date": "2024-01",
    "end_date": "2028-01",
    "sources": [
      "https://www.brightonco.gov/203/City-Council",
      "https://www.brightonco.gov/2872"
    ],
    "updated_at": "2025-06-27T20:53:33Z"
  },
  {
    "name": "Jim Snyder",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 2"
    ],
    "image": "https://www.brightonco.gov/ImageRepository/Document?documentID=29812",
    "phone_number": "303-655-2264",
    "email": "jsnyder@brightonco.gov",
    "website": "https://www.brightonco.gov/708/Jim-Snyder",
    "start_date": "2024-01",
    "end_date": "2028-01",
    "sources": [
      "https://www.brightonco.gov/203/City-Council",
      "https://www.brightonco.gov/708/Jim-Snyder"
    ],
    "updated_at": "2025-06-27T20:53:33Z"
  },
  {
    "name": "Ann Taddeo",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 2"
    ],
    "image": "https://www.brightonco.gov/ImageRepository/Document?documentID=22377",
    "phone_number": "303-655-2266",
    "email": "ataddeo@brightonco.gov",
    "website": "https://www.brightonco.gov/1508/Ann-Taddeo",
    "start_date": "2022-01",
    "end_date": "2026-01",
    "sources": [
      "https://www.brightonco.gov/203/City-Council",
      "https://www.brightonco.gov/1508/Ann-Taddeo"
    ],
    "updated_at": "2025-06-27T20:53:33Z"
  },
  {
    "name": "Chris Fiedler",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 3"
    ],
    "image": "https://www.brightonco.gov/ImageRepository/Document?documentID=29811",
    "phone_number": "303-655-2267",
    "email": "cfiedler@brightonco.gov",
    "website": "https://www.brightonco.gov/1197/Chris-Fiedler",
    "start_date": "2024-01",
    "end_date": "2028-01",
    "sources": [
      "https://www.brightonco.gov/203/City-Council",
      "https://www.brightonco.gov/1197/Chris-Fiedler"
    ],
    "updated_at": "2025-06-27T20:53:33Z"
  },
  {
    "name": "Jan Pawlowski",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 3"
    ],
    "image": "https://www.brightonco.gov/ImageRepository/Document?documentID=28918",
    "phone_number": "303-655-2263",
    "email": "jpawlowski@brightonco.gov",
    "website": "https://www.brightonco.gov/710/Jan-Pawlowski",
    "start_date": "2022-01",
    "end_date": "2026-01",
    "sources": [
      "https://www.brightonco.gov/203/City-Council",
      "https://www.brightonco.gov/710/Jan-Pawlowski"
    ],
    "updated_at": "2025-06-27T20:53:33Z"
  },
  {
    "name": "Lloyd Worth",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 4"
    ],
    "image": "https://www.brightonco.gov/ImageRepository/Document?documentID=29813",
    "phone_number": "303-655-2260",
    "email": "lworth@brightonco.gov",
    "website": "https://www.brightonco.gov/3101/Lloyd-Worth",
    "start_date": "2024-01",
    "end_date": "2028-01",
    "sources": [
      "https://www.brightonco.gov/203/City-Council",
      "https://www.brightonco.gov/3101/Lloyd-Worth"
    ],
    "updated_at": "2025-06-27T20:53:33Z"
  },
  {
    "name": "Peter Padilla",
    "roles": [
      "Mayor Pro Tem",
      "Council Member"
    ],
    "divisions": [
      "Ward 4"
    ],
    "image": "https://www.brightonco.gov/ImageRepository/Document?documentID=28921",
    "phone_number": "303-655-2269",
    "email": "ppadilla@brightonco.gov",
    "website": "https://www.brightonco.gov/724/Peter-Padilla",
    "start_date": "2022-01",
    "end_date": "2026-01",
    "sources": [
      "https://www.brightonco.gov/203/City-Council",
      "https://www.brightonco.gov/724/Peter-Padilla"
    ],
    "updated_at": "2025-06-27T20:53:33Z"
  }
]