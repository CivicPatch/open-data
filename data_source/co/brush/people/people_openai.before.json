[
  {
    "name": "Dana Sherman",
    "roles": [
      "Mayor"
    ],
    "divisions": [
      "Entire City limits"
    ],
    "image": null,
    "phone_number": "970-467-1454",
    "email": "dsherman@brushcolo.com",
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://www.brushcolo.com/your_city/city_council__members.php"
    ],
    "updated_at": "2025-06-30T21:16:25Z"
  },
  {
    "name": "Tyler Horton",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward 1"
    ],
    "image": null,
    "phone_number": "970-768-3229",
    "email": "thorton@brushcolo.com",
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://www.brushcolo.com/your_city/city_council__members.php"
    ],
    "updated_at": "2025-06-30T21:16:25Z"
  },
  {
    "name": "Nathan Tyree",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward I"
    ],
    "image": null,
    "phone_number": "970-441-0299",
    "email": "ntyree@brushcolo.com",
    "website": null,
    "start_date": null,
    "end_date": "2028",
    "sources": [
      "https://www.brushcolo.com/your_city/city_council__members.php"
    ],
    "updated_at": "2025-06-30T21:16:25Z"
  },
  {
    "name": "Scott Klinkhammer",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward II"
    ],
    "image": null,
    "phone_number": "612-581-9537",
    "email": "sklinkhammer@brushcolo.com",
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://www.brushcolo.com/your_city/city_council__members.php"
    ],
    "updated_at": "2025-06-30T21:16:25Z"
  },
  {
    "name": "Daniel R. Scalise",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward II"
    ],
    "image": null,
    "phone_number": "970-768-3971",
    "email": "dscalise@brushcolo.com",
    "website": null,
    "start_date": null,
    "end_date": "2028",
    "sources": [
      "https://www.brushcolo.com/your_city/city_council__members.php"
    ],
    "updated_at": "2025-06-30T21:16:25Z"
  },
  {
    "name": "Larry Lundstrom",
    "roles": [
      "Mayor Pro Tempore"
    ],
    "divisions": [
      "Ward III"
    ],
    "image": null,
    "phone_number": "970-370-3003",
    "email": "llundstrom@brushcolo.com",
    "website": null,
    "start_date": null,
    "end_date": "2026",
    "sources": [
      "https://www.brushcolo.com/your_city/city_council__members.php"
    ],
    "updated_at": "2025-06-30T21:16:25Z"
  },
  {
    "name": "Justin Spradlin",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward III"
    ],
    "image": null,
    "phone_number": "970-380-9371",
    "email": "jspradlin@brushcolo.com",
    "website": null,
    "start_date": null,
    "end_date": "2028",
    "sources": [
      "https://www.brushcolo.com/your_city/city_council__members.php"
    ],
    "updated_at": "2025-06-30T21:16:25Z"
  }
]