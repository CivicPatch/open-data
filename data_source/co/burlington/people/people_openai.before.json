[
  {
    "name": "Greg Swiatkowski",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "719-346-8652",
    "email": "greg.swiatkowski@burlingtoncolo.com",
    "website": "https://www.burlingtoncolo.com/directory.aspx?EID=108",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.burlingtoncolo.com/Directory.aspx?did=18",
      "https://www.burlingtoncolo.com/directory.aspx?EID=108"
    ],
    "updated_at": "2025-06-30T22:03:37Z"
  },
  {
    "name": "Paul Velasco",
    "roles": [
      "Mayor Pro Tem"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "719-346-8652",
    "email": "paul.velasco@burlingtoncolo.com",
    "website": "https://www.burlingtoncolo.com/directory.aspx?EID=94",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.burlingtoncolo.com/Directory.aspx?did=18",
      "https://www.burlingtoncolo.com/directory.aspx?EID=94"
    ],
    "updated_at": "2025-06-30T22:03:37Z"
  },
  {
    "name": "Hal McNerney",
    "roles": [
      "City Council",
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "719-346-8652",
    "email": "hal.mcnerney@burlingtoncolo.com",
    "website": "https://www.burlingtoncolo.com/directory.aspx?EID=79",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.burlingtoncolo.com/Directory.aspx?did=18",
      "https://www.burlingtoncolo.com/directory.aspx?EID=79"
    ],
    "updated_at": "2025-06-30T22:03:37Z"
  },
  {
    "name": "Adrian Hernandez",
    "roles": [
      "City Council",
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "719-346-8652",
    "email": "adrian.hernandez@burlingtoncolo.com",
    "website": "https://www.burlingtoncolo.com/directory.aspx?EID=111",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.burlingtoncolo.com/Directory.aspx?did=18",
      "https://www.burlingtoncolo.com/directory.aspx?EID=111"
    ],
    "updated_at": "2025-06-30T22:03:37Z"
  },
  {
    "name": "Mark Burghart",
    "roles": [
      "City Council",
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "719-346-8652",
    "email": "mark.burghart@burlingtoncolo.com",
    "website": "https://www.burlingtoncolo.com/directory.aspx?EID=76",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.burlingtoncolo.com/Directory.aspx?did=18",
      "https://www.burlingtoncolo.com/directory.aspx?EID=76"
    ],
    "updated_at": "2025-06-30T22:03:37Z"
  },
  {
    "name": "Kamron Weisshaar",
    "roles": [
      "City Council",
      "Council Member"
    ],
    "divisions": [],
    "image": null,
    "phone_number": "719-346-8652",
    "email": "kamron.weisshaar@burlingtoncolo.com",
    "website": "https://www.burlingtoncolo.com/directory.aspx?EID=81",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.burlingtoncolo.com/Directory.aspx?did=18",
      "https://www.burlingtoncolo.com/directory.aspx?EID=81"
    ],
    "updated_at": "2025-06-30T22:03:37Z"
  }
]