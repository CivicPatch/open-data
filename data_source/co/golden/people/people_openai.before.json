[
  {
    "name": "Laura Weinberg",
    "roles": [
      "Mayor"
    ],
    "divisions": [],
    "image": "https://www.cityofgolden.gov/set_site_up/government/2024_COG_LauraWeinberg_jmp_001_sm%20-%20Copy.jpg",
    "phone_number": "(303) 384-8000",
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Lisa Vitry",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "District One"
    ],
    "image": "https://www.cityofgolden.gov/Images/Government/Document%20Center/2024_COG_LisaVitry_jmp_002_sm%20-%20web%20-%20Copy.jpg",
    "phone_number": "(303) 384-8000",
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Patty Evans",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "District Two"
    ],
    "image": "https://www.cityofgolden.gov/Images/Government/Document%20Center/2024_COG_PattyEvans_jmp_002_sm_websquare.jpg",
    "phone_number": "(303) 384-8000",
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Robert Reed",
    "roles": [
      "Council Member",
      "Mayor Pro Tempore"
    ],
    "divisions": [
      "Ward One"
    ],
    "image": "https://www.cityofgolden.gov/Images/Government/City%20leadership/Reed-2022-400x533%20-%20Copy.jpg",
    "phone_number": "(303) 384-8000",
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Paul Haseman",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward Two"
    ],
    "image": "https://www.cityofgolden.gov/Images/Government/City%20leadership/Haseman-2022-400x533%20-%20Copy.jpg",
    "phone_number": "(303) 384-8000",
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Don Cameron",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward Three"
    ],
    "image": "https://www.cityofgolden.gov/Images/Government/City%20leadership/2024_COG_DonCameron_jmp_002_sm-websquare.jpg",
    "phone_number": "(303) 384-8000",
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:43:47Z"
  },
  {
    "name": "Bill Fisher",
    "roles": [
      "Council Member"
    ],
    "divisions": [
      "Ward Four"
    ],
    "image": "https://www.cityofgolden.gov/Images/Government/City%20leadership/2024_COG_BillFisher_jmp_002_sm.jpg",
    "phone_number": "(303) 384-8000",
    "email": null,
    "website": "https://www.cityofgolden.gov/government/city_leadership/city_council.php",
    "start_date": null,
    "end_date": null,
    "sources": [
      "https://www.cityofgolden.net"
    ],
    "updated_at": "2025-06-28T18:43:47Z"
  }
]